tracing-attributes/src/lib.rs

```diff
@@ -385,6 +393,13 @@ fn instrument_precise(
     let input = syn::parse::<ItemFn>(item)?;
     let instrumented_function_name = input.sig.ident.to_string();
 
+    if input.sig.constness.is_some() {
+        return Ok(quote! {
+            compile_error!("instrumenting const fn not supported")
```

Comment:
this feels a little bit terse; what do you think about rephrasing this as 
```suggestion
            compile_error!("the `#[instrument]` attribute may not be used with `const fn`s")
```

Reply:
Great clarity -- thank you!